ci/travis_install_linux.sh

```diff
@@ -37,13 +37,13 @@ fi
 
 set -x
 
-sudo apt-get install -y -qq maven
+#sudo apt-get install -y -qq maven
 
 # Remove Travis-specific versions of Java
-sudo rm -rf /usr/local/lib/jvm*
-sudo rm -rf /usr/local/maven*
-hash -r
-unset JAVA_HOME
+#sudo rm -rf /usr/local/lib/jvm*
```

Comment:
should we instead add a flag to remove this for Xenial and not for trusty..

Reply:
also should we try out a dremio/arrow-build with this patch to catch any other divergences..

Reply:
Done for both.